src/coinjoin/client.h

```diff
@@ -93,7 +94,24 @@ class CoinJoinWalletManager {
 
     CCoinJoinClientManager* Get(const std::string& name) const;
 
-    const wallet_name_cjman_map& raw() const { return m_wallet_manager_map; }
+    template <typename Callable>
+    void ForEachCJClientMan(Callable&& func)
+    {
+        LOCK(cs_wallet_manager_map);
+        for (auto&& [_, clientman] : m_wallet_manager_map) {
+            func(clientman);
+        }
+    };
+
+    template <typename Callable>
+    bool ForAnyCJClientMan(Callable&& func)
+    {
+        LOCK(cs_wallet_manager_map);
+        for (auto&& [_, clientman] : m_wallet_manager_map) {
+            if (func(clientman)) return true;
+        }
+        return false;
```

Comment:
replace with std::any?

Reply:
`std::any` moves otherwise compile-time validation to runtime, it's unclear what the benefit would be here.

Reply:
wym? how does it move compile-time validation to runtime?

Reply:
Because `std::any` is used for type erasure, which introduces runtime overhead. Replaced with `ranges::any_of` as suggested in 019a78033448fa97dc11ef1308f1f29e6e021013.

Reply:
Yeah, I meant any_of